scripts/cleanup_old_vms.py

```diff
@@ -188,7 +193,7 @@ def scan_vm(provider_key, vm, delta, match_queue, scan_failure_queue):
         logger.info('%r: VM %r did not match age requirement', provider_key, vm.name)
 
 
-def delete_vm(provider_key, vm, age, result_queue):
```

Comment:
**REQUIRED:** These seem to be going against the new model of always passing in the objects, I know there must be a reason for this, can you explain it a little?

Reply:
Yep - the objects being passed into the function are done through the pool_manager, and have to be pickled. The wrapanapi VM objects aren't pickle-able.  I know I was hitting it on trying to get the namedtuple results, and changed them going into the function as well, but I'll double check that behavior.

Reply:
Part of the reasoning is that this script is running independent of any tests, and the VM could have been deleted by tests or users between scanning the provider, scanning the vm, and trying to delete.  So each time we call `get_vm` on it to fetch it, though this is secondary and we could have vm.exists traps.